clang/lib/Sema/SemaChecking.cpp

```diff
@@ -1981,6 +1981,26 @@ static bool OpenCLBuiltinToAddr(Sema &S, unsigned BuiltinID, CallExpr *Call) {
   return false;
 }
 
+// In OpenCL, __builtin_alloca_* should return a pointer to address space
+// that corresponds to the stack address space i.e private address space.
+static bool OpenCLBuiltinAllocaAddrSpace(Sema &S, CallExpr *TheCall) {
+  QualType RT = TheCall->getType();
+  if (!RT->isPointerType() || RT->getPointeeType().hasAddressSpace())
+    return true;
+
+  if (S.getLangOpts().OpenCL) {
+    RT = RT->getPointeeType();
+
+    // Stack Address space corresponds to private address space.
+    LangAS openCLStackAS = LangAS::opencl_private;
```

Comment:
Don't need this variable or name, just directly pass LangAs::opencl_private to getAddrSpaceQualType

Reply:
Done